sklearn/tree/_classes.py

```diff
@@ -546,7 +546,8 @@ def cost_complexity_pruning_path(self, X, y, sample_weight=None):
         Returns
         -------
         ccp_path : Bunch
```

Comment:
what happens if we do it as:

```suggestion
        ccp_path : :class:`~sklearn.utils.Bunch`
```

Would it render properly with the link? If yes, then we won't need the `For more information, see :class:`sklearn.utils.Bunch` part.

Reply:
It can render properly, so I fixed it. Thank you for review :)